### PR TITLE
ClusterLoader - Latency pod memory update - 100 nodes

### PR DESCRIPTION
**Pull request by krzysied**
Using better memory estimation for latency pods in gce 100 test. This will save ~200MB for potential probe pods.

Scheduler uses allocatable memory to evaluate node score. Nodes have < 3500MB memory available. Therefore 350MB should be a good choice.

### CODE REVIEW

File: clusterloader2/testing/density/100_nodes/override.yaml [hunk 1]
```diff
@@ -1 +1,2 @@
 DENSITY_RESOURCE_CONSTRAINTS_FILE: ./100_nodes/constraints.yaml
+LATENCY_POD_MEMORY: 350
```
Comment (wojtek-t) [inline clusterloader2/testing/density/100_nodes/override.yaml]: Why can't we change it everywhere? What's the point of introducing differences between tests?

Comment (krzysied) [reply clusterloader2/testing/density/100_nodes/override.yaml]: You're right. I'm changing density test default.

**Comment by mm4tt** [comment]
Could you check why the tests failed?

**Comment by krzysied** [comment]
Tests failed with `ERROR: (gcloud.compute.routers.create) Quota 'ROUTERS' exceeded. Limit: 10.0 globally.` during `up` phase.
I will check project quota.

**Comment by krzysied** [comment]
Resources are available in the project, so it seems it was a random collision with other tests (I guess).

**Comment by krzysied** [comment]
TL; DR Reason of test failure was unrelated to changes introduced in this PR.
